database/buildSchema/41_application_list_view.sql

```diff
@@ -62,6 +65,9 @@ CREATE FUNCTION application_list (userid int DEFAULT 0)
     LEFT JOIN assignment_list (stage_status.stage_id) ON app.id = assignment_list.application_id
     LEFT JOIN review_list (stage_status.stage_id, $1) ON app.id = review_list.application_id
     LEFT JOIN assigner_list (stage_status.stage_id, $1) ON app.id = assigner_list.application_id
+    LEFT JOIN trigger_schedule ts ON app.id = ts.application_id
```

Comment:
If we misconfigure actions and there are multiple rows with that event_code, application list would duplicates, just something to keep in mind (all the other joins are meant to return just one row)

Reply:
No, I have a joint unique constraint on the trigger_schedule table for "application_id" and "event_code", so that can't happen ;)

```
CREATE UNIQUE INDEX unique_application_event ON trigger_schedule (application_id, event_code);
```